test/llvmcall.jl

```diff
@@ -48,3 +48,44 @@ baremodule PlusTest
     end
     @test Int32(1)+Int32(2)==Int32(3)
 end
+
+# Test whether declarations work properly
+function undeclared_ceil(x::Float64)
+    llvmcall("""%2 = call double @llvm.ceil.f64(double %0)
+        ret double %2""", Float64, Tuple{Float64}, x)
+end
+@test_throws ErrorException undeclared_ceil(4.2)
```

Comment:
That I really doubt... But maybe.....
The GC root needs to be fix though, unless I'm missing sth.